src/main/java/teammates/ui/template/CourseEditSectionRow.java

```diff
@@ -0,0 +1,225 @@
+        String onClick = "";
+        if (!isSessionsInSectionSpecial()) {
+            content = "Give different permissions for sessions in this section";
+            onClick = "showTuneSessionnPermissionsDiv(" + instructorIndex + ", " + sectionIndex + ")";
+        } else {
+            content = "Hide session-level permissions";
+            onClick = "hideTuneSessionnPermissionsDiv(" + instructorIndex + ", " + sectionIndex + ")";
+        }
+        
+        String id = "toggleSessionLevelInSection" + sectionIndex + "ForInstructor" + instructorIndex;
+        toggleSessionLevelInSectionButton = createButton(content, "small col-sm-5", id, "javascript:;", null,
+                                                         onClick, false);
+        
+        for (String feedbackName : feedbackNames) {
+            CourseEditFeedbackSessionRow feedbackSessionRow = new CourseEditFeedbackSessionRow(feedbackName);
+            
+            String name = Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS 
+                          + Const.ParamsNames.INSTRUCTOR_SECTION_GROUP + sectionIndex + "feedback" 
+                          + feedbackName;
+            boolean isChecked = (instructor != null) 
+                                && instructor.isAllowedForPrivilege(sectionName, feedbackName, 
+                                                Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS);
+            ElementTag submitSessionInSectionCheckBox = createCheckBox(null, name, "true", isChecked);
+            
+            name = Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS 
+                   + Const.ParamsNames.INSTRUCTOR_SECTION_GROUP + sectionIndex + "feedback" + feedbackName;
+            isChecked = (instructor != null) 
+                        && instructor.isAllowedForPrivilege(sectionName, feedbackName, 
+                                        Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS);
+            ElementTag viewSessionInSectionCheckBox = createCheckBox(null, name, "true", isChecked);
+            
+            name = Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS 
+                   + Const.ParamsNames.INSTRUCTOR_SECTION_GROUP + sectionIndex + "feedback" + feedbackName;
+            isChecked = (instructor != null)
+                        && instructor.isAllowedForPrivilege(sectionName, feedbackName, 
+                                        Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS);
+            ElementTag modifySessionInSectionCheckBox = createCheckBox(null, name, "true", isChecked);
+            
+            feedbackSessionRow.getPermissionCheckBoxes().add(submitSessionInSectionCheckBox);
+            feedbackSessionRow.getPermissionCheckBoxes().add(viewSessionInSectionCheckBox);
+            feedbackSessionRow.getPermissionCheckBoxes().add(modifySessionInSectionCheckBox);
```

Comment:
Like earlier, possible to make a method-scoped list and toss it into the constructor?